interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -1215,3 +1215,29 @@ def test_motz_wise(self):
 
         # M&W toggled on (locally) for reaction 9
         self.assertNear(2.0 * k1[9], k2[9]) # sticking coefficient = 1.0
+
+class TestButlerVolmer(utilities.CanteraTest):
+
+    def test_pressure_dependence(self):
```

Comment:
A better name for this might be `test_rate_pressure_independence` to clarify what it's actually testing

Reply:
Good idea.  Done.